lib/Settings/routes.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:cboard_mobile/Settings/About.dart';
+import 'package:cboard_mobile/Settings/Feedback.dart';
+import 'package:cboard_mobile/Settings/Navigation.dart';
+import 'package:cboard_mobile/Settings/Speech.dart';
+import 'package:cboard_mobile/Settings/User_help.dart';
+
+import 'Display.dart';
+
+getRoute() {
+  return {
+    '/display': (context) => Display(),
+    '/speech': (context) => Speech(),
+    '/navigation': (context) => Navigation(),
+    '/user help': (context) => User_help(),
+    '/about': (context) => About(),
```

Comment:
@zachmothner @darrenzhang2000 - Here, we're using a `routes.dart` file that renders HTML endpoints. That's different from what we have in the repository so far, so what's the general consensus on how we should implement routing?

Reply:
Just a quick comment, I wrap every features like `display` `speech` ... in a list and I used same code to render widget for them so I think it faster if I used this way of routing

Reply:
Instead of using URLs, can you use the render the widget directly instead? That's what I've been using, but I'm not sure which one is more preferable 😅 !

Reply:
I can if you all want. But is it that if we use the `route` file, the code is cleaned up a bit ?

Reply:
Ahh, I totally understand that! It's the same thing either way, but did @zachmothner or @darrenzhang2000 have a preference of which one to use?